app/assets/stylesheets/common/base/discourse.scss

```diff
@@ -274,8 +274,10 @@ select {
   border: 1px solid var(--primary-low);
 }
 
-table {
-  th {
+table,
+.topic-list {
```

Comment:
Nitpick, but can we use this here: 

```
table th, 
.topic-list .topic-list-header {
...
}
```

In the compiled CSS it would result in fewer lines and no mix-and-matching.

Reply:
👍